src/hooks/useWindowDimensions.ts

```diff
@@ -15,6 +15,8 @@ const getWindowDimensions = () => {
 	const is2KDesktop = width > 1920 && width <= 2560;
 	const is4KDesktop = width > 2560;
 
+	const isMobileDevice = isIOS || isMobile;
```

Comment:
instead of isMobileDevice use 
`const isMobile = width <= 640 || isIOS;`
so and remove `isMobileDevice`

Reply:
Done